src/lib/temple/front/assets.ts

```diff
@@ -218,25 +218,33 @@ export const useAvailableRoute3Tokens = () => {
   };
 };
 
-const FIRST_TOKENS = [TEZ_TOKEN_SLUG, TEMPLE_TOKEN_SLUG];
+const FIRST_HOME_PAGE_TOKENS = [TEZ_TOKEN_SLUG, TEMPLE_TOKEN_SLUG];
+const FIRST_SWAP_SEND_TOKENS = [TEZ_TOKEN_SLUG];
 
-function makeAssetsSortPredicate(balances: Record<string, BigNumber>, fiatToTokenRates: Record<string, string>) {
+function makeAssetsSortPredicate(
+  balances: Record<string, BigNumber>,
+  fiatToTokenRates: Record<string, string>,
+  leadingAssetsSlugs?: Array<string>
+) {
   return (tokenASlug: string, tokenBSlug: string) => {
-    const tokenAIncluded = FIRST_TOKENS.includes(tokenASlug);
-    const tokenBIncluded = FIRST_TOKENS.includes(tokenBSlug);
+    if (isDefined(leadingAssetsSlugs)) {
+      const tokenAIncluded = leadingAssetsSlugs.includes(tokenASlug);
```

Comment:
Please refactor this code to avoid nested conditionals because of checking whether an array is defined. Using empty array as a default value should work well